### ISSUE TITLE
Plugin not detected, just says undefined

### ISSUE THREAD

**Issue by syn6UK** [transistorsoft/cordova-plugin-background-fetch#96]
## Your Environment
* Plugin version: Latest
* Platform: iOS or Android IOS
* OS version: 11
* Device manufacturer / model: IphoneX
* Cordova version (`cordova -v`): Latest
* Cordova platform version (`cordova platform ls`): Latest
* Plugin config

## Expected Behavior

Copied your ionic 2+ example

## Actual Behavior

It fails on     let BackgroundFetch = (<any>window).BackgroundFetch;

When I dump that variable it says null. It's as if it's not acknowleding the plugin.

## Steps to Reproduce

1. ionic start projectname tabs
2. ionic cordova plugin add cordova-plugin-background-fetch 
3. copy ionic 2 example into the home.ts file
4. Try and run the app using Ionic devapp

## Debug logs

Console log shows this

[13:53:13]  console.warn: Native: tried calling StatusBar.styleDefault, but Cordova is not available. Make sure to 
            include cordova.js or run in a device/simulator 
[13:53:13]  console.warn: Native: tried calling SplashScreen.hide, but Cordova is not available. Make sure to include 
            cordova.js or run in a device/simulator 
[13:53:13]  console.log: null

Which makes me think its not acknowledging cordova at all but it is in there.

**Comment by MurmeltierS** [comment]
I seem to have a similar problem, any fixes?

**Comment by syn6UK** [comment]
Nope sorry :(

Rob Lane
Lead Designer / Director



IMPORTANT - WARNING AND CONFIDENTIALITY NOTICE

This e-mail does not form the basis of a contract unless specifically
stated.
This e-mail message and any files transmitted with it are confidential and
intended solely for the use of the individual or entity to which it is
addressed. If you have received this e-mail in error or the message is
received by anyone other than the addressee, please notify me by telephone
or return the e-mail and then delete all copies from your computer system.
Internet e-mails are not necessarily secure. I do not accept responsibility
for changes made to this message after it was sent.
Whilst all reasonable care has been taken to avoid the transmission of
viruses, it is the responsibility of the recipient to ensure that the
onward transmission, opening or use of this message will not adversely
affect its systems or data.

**Comment by crawft** [comment]
Same here, can't see BackgroundFetch in any of the likely places - window, window.cordova, window.cordova.plugins etc. I did it this way because the ionic native wrapped version still says it's iOS only but I might try that...

**Comment by christocracy** [comment]
@crawft There's no need to randomly hunt-and-peck for the interface.  The location of the API is documented right there in the [README](https://github.com/transistorsoft/cordova-plugin-background-fetch#using-the-plugin)

`window.BackgroundFetch`

Don't use the Ionic wrapper.  It's out-dated.

If using Ionic 2+, follow those directions in the [README](https://github.com/transistorsoft/cordova-plugin-background-fetch#ionic-2-example)

**Comment by crawft** [comment]
In version 5.4.1, the plugin does not exist at window.BackgroundFetch, that's why this issue has been raised. People have essentially copied and pasted the code in the  README, and it doesn't work. I realised I started seeing this after upgrading the plugin, so for now I've reverted to 5.3.0 (which is also the version in the ionic native wrapper), I believe that in v5.3.0, the plugin DID exist at window.BackgroundFetch.
